server/src/domain/job/job.constants.ts

```diff
@@ -73,6 +73,7 @@ export enum JobName {
   // cleanup
   DELETE_FILES = 'delete-files',
   CLEAN_OLD_AUDIT_LOGS = 'clean-old-audit-logs',
+  MOVE_HISTORY_CLEANUP = 'move-history-cleanup',
```

Comment:
So I am wondering if we want to use this table for both active records _and_ an audit, vs just active records. It might be easier to use it for active records with unique key constraints and remove the soft delete stuff. We could always track asset updates in another "changes" table, which could include things in addition to path updates.